docs/modules/ROOT/pages/backup-restore.adoc

```diff
@@ -107,77 +128,100 @@ kubectl create secret generic <secret-name> \
+To restore a cluster from a local backup, you can choose from the following approaches:
 
+* Reapply the Hazelcast custom resource directly. 
++
+This approach gives the cluster access to the PVCs that contain the persisted data. The Hazelcast cluster is restored from the existing `hot-restart` folders.
 
+* Restore from a local backup, which was created using the HotBackup resource. 
++
+To use this approach, you must have configured `hotBackupResource Name` in the `restore` section of your Hazelcast custom resource configuration. For the restore to work correctly, the specified `HotBackup` resource must have been created successfully. You can check the status of your local backup as described in <<scheduling-backups,Scheduling Backups>>
++
+For this approach, the `platform-operator-agent` container, also known as the sidecar agent and Agent, is
+deployed with the Hazelcast container in the same Pod. Agent starts as an init container, which is a specialized container that runs before the Hazelcast container.
++
+The configuration is as follows:
++
 [source,yaml,subs="attributes+"]
 ----
 include::ROOT:example$/hazelcast-persistence-restore-cr-name.yaml[]
 ----
 
-<1> `HotBackup` resource name used for restore. The backup folder name will be taken from the `HotBackup` resource.
-<2> Agent responsible for restoring data from the local storage.
-    The agent configuration is optional. If you give `restore` under `persistence` and do not pass the agent configuration, Hazelcast Platform Operator
-    uses the latest agent version that is compatible with its version.
+<1> `HotBackup` resource name used for restore. The backup folder name is taken from the `HotBackup` resource.
+<2> Optional. This is always `hazelcast/platform-operator-agent`. 
+Agent responsible for restoring data from local storage and cleaning up after the move. 
+If you enable `persistence` and do not pass the agent configuration, Hazelcast Operator uses the latest compatible version of agent.
+
+[restoring-from-external-backups]
+== Restore from External Backups
 
-== Restoring from External Backups
+To restore a backed-up cluster from an external bucket, you can configure either:
 
-To restore a cluster from external backups, you can either set up the bucket configuration or give the `HotBackup` resource name that you used to trigger the external backup. In either case, the backup is restored from the external bucket.
+* The URI and secret of the bucket.
+* The name of the HotBackup resource that was used to trigger the external backup.
 
-When this restore mechanism is used, the Restore Agent container is deployed with the Hazelcast container in the same Pod. The agent starts as an initContainer before the Hazelcast container.
+The `platorm-operator-agent` container is deployed with the Hazelcast container in the same Pod. The agent starts as an init container, also known as the sidecar agent and Agent, which is a specialized container that runs before the Hazelcast container.
```

Comment:
```suggestion
For this approach, the `platform-operator-agent` container is deployed as an init container along with the Hazelcast container in the same Pod. The agent starts before the Hazelcast container to download backup folders from the source bucket and arrange them in case of a successful restore.
```

Reply:
Updated